app/src/main/res/layout/add_vocab.xml

```diff
@@ -120,7 +120,7 @@
         android:layout_marginEnd="128dp"
         android:layout_marginStart="127dp"
         android:layout_marginTop="8dp"
-        android:text="save Translation"
+        android:text="x"
```

Comment:
Why is that?
Explanation or correction please

Reply:
Thanks, I solved it :)